apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedTabs/EmailTab.tsx

```diff
@@ -365,6 +366,42 @@ const getEmailTemplate = (survey: TSurvey, surveyUrl: string, brandColor: string
           <EmailFooter />
         </EmailTemplateWrapper>
       );
+    case QuestionType.Welcome:
+      return (
+        <EmailTemplateWrapper surveyUrl={url} brandColor={brandColor}>
+          {firstQuestion.companyLogo && (
+            <img
```

Comment:
Please use the Image component from next/image instead of html img component

Reply:
Done